capellambse_context_diagrams/context.py

```diff
@@ -394,7 +421,9 @@ def _create_diagram(self, params: dict[str, t.Any]) -> cdiagram.Diagram:
         )
         data, legend = tree_view.collector(self, params)
         params["elkdata"] = data
-        class_diagram = super()._create_diagram(params)
+        layout = _elkjs.call_elkjs(data)
+        add_context(layout)
```

Comment:
`add_context` is just called in the `ClassTreeDiagram` creator function. Don't we need context for all context diagram types? Please add context in all of the diagrams.